ambry-api/src/test/java/com.github.ambry/router/GetBlobOptionsTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.router;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+
+public class GetBlobOptionsTest {
+  @Test
+  public void testRangeOption()
+      throws Exception {
+    ByteRange range = ByteRange.fromOffsetRange(1, 2);
+    GetBlobOptions options = new GetBlobOptions(range);
+    assertEquals(range, options.getRange());
```

Comment:
Also, it looks like you might to define an equals function in `ByteRange`. It isn't wrong for `GetBlobOptions` to create a new `ByteRange` object. So what you really want to be checking here is not that it is the same reference, but is the same values and type. Right?